src/app/coursegrab/utils/scraper.py

```diff
@@ -12,6 +12,17 @@
 def start_update():
     global on_startup
     try:
+        print("[{0}] Checking for new semester".format(datetime.now()))
+        tracking = semesters_dao.get_current_semester()
+        current = current_semester()
+
+        # On app startup or new semester in roster
+        if tracking is None or tracking["current_semester"] != current:
+            print("[{0}] Clearing information from previous semester".format(datetime.now()))
+            courses_dao.clear_table()
+            sections_dao.clear_table()
+            semesters_dao.update_current_semester(current)
```

Comment:
maybe we should move this into the `refresh_classes` method since that runs once a day versus `start_update` runs every 5 minutes